lib/improver/tests/ensemble_calibration/ensemble_calibration/test_ApplyCoefficientsForEnsembleCalibration.py

```diff
@@ -271,245 +275,169 @@ def test_unmatching_y_axis(self):
+        calculated when using the ensemble realizations with statsmodels.
+        These expected values are compared to the results when using the
+        ensemble realizations to ensure that the results are similar."""
         optimised_coeffs = dict(
             zip(self.coeffs_from_statsmodels_realizations.coord(
                     "coefficient_name").points,
                 self.coeffs_from_statsmodels_realizations.data))
-        expected_forecast_predictors = (
-            self.current_temperature_forecast_cube.collapsed(
-                "realization", iris.analysis.MEAN))
-        predicted_mean, forecast_predictors = (
-            self.plugin._get_calibrated_forecast_predictors_realizations(
-                optimised_coeffs, self.forecast_vars))
+        location_parameter = (
+            self.plugin._calculate_location_parameter_from_realizations(
+                optimised_coeffs))
         self.assertCalibratedVariablesAlmostEqual(
-            predicted_mean,
-            self.expected_calibrated_predictor_statsmodels_realizations)
-        self.assertCalibratedVariablesAlmostEqual(
-            forecast_predictors.data, expected_forecast_predictors.data)
+            location_parameter,
+            self.expected_loc_param_statsmodels_realizations)
+        assert_array_almost_equal(
+            location_parameter,
+            self.expected_loc_param_mean,
+            decimal=0)
+        assert_array_almost_equal(
+            location_parameter,
+            self.expected_loc_param_no_statsmodels_realizations,
+            decimal=0)
 
     @ManageWarnings(
         ignored_messages=["Collapsing a non-contiguous coordinate."])
     def test_without_statsmodels(self):
-        """
-        Test that the expected values are returned by this function when not
-        using statsmodels.
-        """
+        """Test that the expected values for the location parameter are
+        calculated when using the ensemble realizations without statsmodels.
+        These expected values are compared to the results when using the
+        ensemble realizations to ensure that the results are similar."""
         optimised_coeffs = dict(
             zip(self.coeffs_from_no_statsmodels_realizations.coord(
                     "coefficient_name").points,
                 self.coeffs_from_no_statsmodels_realizations.data))
-        expected_forecast_predictors = (
-            self.current_temperature_forecast_cube.collapsed(
-                "realization", iris.analysis.MEAN))
-        predicted_mean, forecast_predictors = (
-            self.plugin._get_calibrated_forecast_predictors_realizations(
-                optimised_coeffs, self.forecast_vars))
-        self.assertCalibratedVariablesAlmostEqual(
-            predicted_mean,
-            self.expected_calibrated_predictor_no_statsmodels_realizations)
+        location_parameter = (
+            self.plugin._calculate_location_parameter_from_realizations(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Access to a protected member _calculate_location_parameter_from_realizations of a client class](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=4714944)